pyspecProcScripts/integral_w_error.py

```diff
@@ -43,23 +44,25 @@ def integral_w_errors(s,sig_path,error_path, indirect='vd', direct='t2',fl=None,
              +str(extra_dims))
     collected_variance = ndshape(
          [ndshape(s)[indirect],len(error_path)],[indirect,'pathways']).alloc()
+    # this is not averaging over all the pathways!!!!
     for j in range(len(error_path)):
-     # calculate N₂ Δf² σ², which is the variance of the integral (by error propagation)
-     # where N₂ is the number of points in the indirect dimension
-     s_forerror = select_pathway(s,error_path[j])
-     if j==0: N2 = ndshape(s_forerror)[direct]
-     # mean divides by N₁ (indirect), integrate multiplies by Δf, and the
-     # mean sums all elements (there are N₁N₂ elements)
-     s_forerror -= s_forerror.C.mean_all_but([indirect, direct]).mean(direct)
```

Comment:
I do not understand how you end up w/ for loops that are only indented by one space?  you should just be able to hit tab in vi to get the correct number of spaces, if all your settings are correct.

Reply:
it's indented by a single space -- Python understands that, but humans do not :-)